credentials/test/gen-test-paa-trust-store-list.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+
+#
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import sys
+from cryptography.hazmat.primitives import serialization
+from cryptography import x509
+
+paa_trust_store_list = []
+
+for subdir, dirs, files in os.walk(str(sys.argv[1])):
+    for file in files:
+        if file.startswith(str(sys.argv[2])) and file.endswith(str(sys.argv[3])):
+            with open(str(sys.argv[1]) + '/' + file, 'rb') as infile:
+                paa_trust_store_list.append(
+                    x509.load_pem_x509_certificate(infile.read()))
+
+paa_trust_store_list_bytes = []
+
+for paa_bytes in paa_trust_store_list:
+    paa_trust_store_list_bytes.append(
+        paa_bytes.public_bytes(serialization.Encoding.DER))
+
+with open(str(sys.argv[4]), 'wb') as outfile:
+    for paa_bytes in paa_trust_store_list_bytes:
+        outfile.write(len(paa_bytes).to_bytes(4, 'little'))
+        outfile.write(paa_bytes)
+        outfile.write(b"\n")
```

Comment:
This format here is non-trivial. As discussed offline, let's move to a model where we get a path, glob it and everything matching is the list of PAAs loaded at runtime